assets/js/theme/custom/order-product-data.js

```diff
@@ -0,0 +1,55 @@
+/* @TODO: Need to figure out a way to hide API TOKEN on the front. */
+export default function getOrderProductData(apiToken) {
+    const selectors = {
+        item: '.js-card-order',
+        url: '.js-dynamic-url',
+        wishlist: '.js-wishlist-add',
+    }
+
+    const items = Array.from(document.querySelectorAll(selectors.item))
+    items.forEach((item) => {
+        const links = Array.from(item.querySelectorAll(selectors.url))
+        const sku = item.getAttribute('data-sku')
+        const wishlistForm = item.querySelector(selectors.wishlist)
+
+        fetch('/graphql', {
+            method: 'POST',
+            credentials: 'same-origin',
+            headers: {
+                'Content-Type': 'application/json',
+                Authorization: `Bearer ${apiToken}`,
+            },
+            body: JSON.stringify({
+                query: `
+                        query myQuery {
```

Comment:
Please rename `query myQuery` to a proper name for what the query is supposed to do